[2025-06-18 15:03:46,339] {taskinstance.py:851} INFO - Dependencies all met for <TaskInstance: first_dag.third_task 2023-01-20T00:00:00+00:00 [queued]>
[2025-06-18 15:03:46,379] {taskinstance.py:851} INFO - Dependencies all met for <TaskInstance: first_dag.third_task 2023-01-20T00:00:00+00:00 [queued]>
[2025-06-18 15:03:46,380] {taskinstance.py:1042} INFO - 
--------------------------------------------------------------------------------
[2025-06-18 15:03:46,380] {taskinstance.py:1043} INFO - Starting attempt 1 of 6
[2025-06-18 15:03:46,380] {taskinstance.py:1044} INFO - 
--------------------------------------------------------------------------------
[2025-06-18 15:03:46,438] {taskinstance.py:1063} INFO - Executing <Task(BashOperator): third_task> on 2023-01-20T00:00:00+00:00
[2025-06-18 15:03:46,447] {standard_task_runner.py:52} INFO - Started process 8374 to run task
[2025-06-18 15:03:46,461] {standard_task_runner.py:76} INFO - Running: ['airflow', 'tasks', 'run', 'first_dag', 'third_task', '2023-01-20T00:00:00+00:00', '--job-id', '11279', '--pool', 'default_pool', '--raw', '--subdir', 'DAGS_FOLDER/first_dag.py', '--cfg-path', '/tmp/tmpp3fkq_0m', '--error-file', '/tmp/tmpgysaf0r_']
[2025-06-18 15:03:46,471] {standard_task_runner.py:77} INFO - Job 11279: Subtask third_task
[2025-06-18 15:03:46,625] {logging_mixin.py:104} INFO - Running <TaskInstance: first_dag.third_task 2023-01-20T00:00:00+00:00 [running]> on host 500dc4ac3a72
[2025-06-18 15:03:46,835] {taskinstance.py:1257} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=my_name
AIRFLOW_CTX_DAG_ID=first_dag
AIRFLOW_CTX_TASK_ID=third_task
AIRFLOW_CTX_EXECUTION_DATE=2023-01-20T00:00:00+00:00
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2023-01-20T00:00:00+00:00
[2025-06-18 15:03:46,850] {bash.py:135} INFO - Tmp dir root location: 
 /tmp
[2025-06-18 15:03:46,856] {bash.py:158} INFO - Running command: echo "this is third task "
[2025-06-18 15:03:46,881] {bash.py:169} INFO - Output:
[2025-06-18 15:03:46,884] {bash.py:173} INFO - this is third task
[2025-06-18 15:03:46,885] {bash.py:177} INFO - Command exited with return code 0
[2025-06-18 15:03:47,081] {taskinstance.py:1166} INFO - Marking task as SUCCESS. dag_id=first_dag, task_id=third_task, execution_date=20230120T000000, start_date=20250618T150346, end_date=20250618T150347
[2025-06-18 15:03:47,212] {taskinstance.py:1220} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2025-06-18 15:03:47,286] {local_task_job.py:146} INFO - Task exited with return code 0
